File: rest/sip-in/create-acl-address/create-acl-address.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$ipAddress = $client->account->sip
    ->ipAccessControlLists("AL32a3c49700934481addd5ce1659f04d2")
    ->ipAddresses
    ->create("My office IP Address", "55.102.123.124");

echo $ipAddress->sid;

File: rest/sip-in/create-acl-address/create-acl-address.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

ip_address = client.sip \
    .ip_addresses("AL32a3c49700934481addd5ce1659f04d2") \
    .create(friendly_name="My office IP Address",
            ip_address="55.102.123.124")
print(ip_address.sid)
